### ISSUE TITLE
Add application to V6 API

### ISSUE THREAD

**Issue by jaredlockhart** [mozilla/experimenter#4146]
We need to expose the `application` field in the V6 API for jetstream to do its telemetry filtering effectively for a given application, so let's add that field.

Blocked on https://github.com/mozilla/nimbus-shared/issues/145

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-672)
┆Issue Number: EXP-672

**Comment by tdsmith** [comment]
Thanks for opening. An application field is defined in the Experiment DTO: https://github.com/mozilla/nimbus-shared/blob/main/types/experiments.ts#L21-L22

We need to
- define a list of allowed values for application
- add a new field to the DTO to capture the release channel, because Jetstream needs to know this for Glean apps

**Comment by jaredlockhart** [comment]
Oh this is already done.

We can add an API endpoint for application here: https://github.com/mozilla/experimenter/issues/4147
And expose channel here: https://github.com/mozilla/experimenter/issues/4148
